types/circular-dependency-plugin/index.d.ts

```diff
@@ -13,6 +13,8 @@ export = CircularDependencyPlugin;
  */
 declare class CircularDependencyPlugin extends Plugin {
```

Comment:
I think Webpack 5 doesn't export `Plugin` anymore, might as well just drop it. I searched DT for inspiration, and looks like it's either `implements WebpackPluginInstance`, or nothing at all, since [`WebpackPluginInstance` just says "yup, it has an apply"](https://github.com/webpack/webpack/blob/45829826701ff6582ea16bc36376d105ff264c26/types.d.ts#L11954-L11961). But probably nicer to be explicit and do an `extends`. Your pick.

Reply:
Thanks! I've changed it to `implements WebpackPluginInstance`